source/frontend/src/features/research/list/ResearchListView.tsx

```diff
@@ -64,7 +65,7 @@ export const ResearchListView: React.FunctionComponent<React.PropsWithChildren<u
   return (
     <Styled.ListPage>
       <Styled.Scrollable>
-        <Styled.PageHeader>Research Files</Styled.PageHeader>
+        <CommonStyled.H1>Research Files</CommonStyled.H1>
```

Comment:
The story in JIRA calls for adding the corresponding icons to each "manage" page but I see no icons here. Was this missed from implementation or did the story requirements changed?

![image](https://github.com/user-attachments/assets/5e674e20-fb36-4253-90ff-7223cbfd08ce)

Reply:
good catch. I've updated the list view titles with the icons